docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -244,4 +244,1052 @@ You have a general idea of what nf-core offers and you know how to run an nf-cor
+[['id':'SAMPLE3_SE', 'sequencer':'sequencer3', 'single_end':true], ... ]
+```
+
+We can comment the `ch_samplesheet.view()` line or remove it. We are not going to use it anymore in this training section.
+
+### Use the new meta key in the pipeline
+
+We can access this new meta value in the pipeline and use it to, for example, only enable trimming for samples from a particular sequencer. The [branch operator](https://www.nextflow.io/docs/stable/reference/operator.html#branch) let's us split
+an input channel into several new output channels based on a selection critera:
+
+```groovy title="workflows/myfirstpipeline.nf" linenums="35"
+ch_seqtk_in = ch_samplesheet.branch { meta, reads ->
+    to_trim: meta["sequencer"] == "sequencer2"
+    other: true
+}
+
+SEQTK_TRIM (
+    ch_seqtk_in.to_trim
+)
+```
+
+If we now rerun our default test, no reads are being trimmed (even though we did not specify `--skip_trim`):
+
+```console
+nextflow run . -profile docker,test --outdir results
+
+[-        ] process > MYORG_MYFIRSTPIPELINE:MYFIRSTPIPELINE:SEQTK_TRIM          -
+[52/7ac3d2] process > MYORG_MYFIRSTPIPELINE:MYFIRSTPIPELINE:FASTQE (SAMPLE1_PE) [100%] 3 of 3 ✔
+[5a/f580bc] process > MYORG_MYFIRSTPIPELINE:MYFIRSTPIPELINE:MULTIQC             [100%] 1 of 1 ✔
+```
+
+If we use the samplesheet with the `sequencer` set, only one sample will be trimmed:
+
+```console
+nextflow run . -profile docker,test --outdir results --input ../data/sequencer_samplesheet.csv -resume
+
+[47/fdf9de] process > MYORG_MYFIRSTPIPELINE:MYFIRSTPIPELINE:SEQTK_TRIM (SAMPLE2_PE) [100%] 1 of 1 ✔
+[f1/e8d6fa] process > MYORG_MYFIRSTPIPELINE:MYFIRSTPIPELINE:FASTQE (SAMPLE1_PE)     [100%] 3 of 3 ✔
+[2a/a742ae] process > MYORG_MYFIRSTPIPELINE:MYFIRSTPIPELINE:MULTIQC                 [100%] 1 of 1 ✔
+```
+
+### Takeaway
+
+You know how to adapt the samplesheet to add new meta information to your files.
+
+### What's next?
+
+In the next step we will add a module that is not yet in nf-core.
+
+---
+
+## Create a custom module for your pipeline
+
+nf-core offers a comprehensive set of modules that have been created and curated by the community. However, as a developer, you may be interested in bespoke pieces of software that are not apart of the nf-core repository or customizing a module that already exists.
+
+In this instance, we will write a local module for the QC Tool [FastQE](https://fastqe.com/), which computes stats for FASTQ files and print those stats as emoji.
+
+This section should feel familiar to the `hello_modules` section.
+
+### Create the module
+
+!!! note "New module contributions are always welcome and encouraged!"
+
+    If you have a module that you would like to contribute back to the commmunity, reach out on the nf-core slack or open a pull request to the modules repository.
+
+Start by using the nf-core tooling to create a kceleton local module. It will prompt you to type in the tool name `fastqe`, for the remaining fields press `enter` to accept the default:
```

Comment:
I had a feeling an error is thrown if the module already exists. I'm not sure if this is still the case but it might be worth noting if it is.

Reply:
good point! Added a warning